CHANGELOG.md

```diff
@@ -2,206 +2,120 @@
-* Feature:  Made various improvements to the onboarding wizard (#1772) ([20dd70b](https://github.com/zowe/api-layer/commit/20dd70b)), closes [#1772](https://github.com/zowe/api-layer/issues/1772)
-
-
-* Bugfix:  Caching service logging (#2222) ([5ff64d9](https://github.com/zowe/api-layer/commit/5ff64d9)), closes [#2222](https://github.com/zowe/api-layer/issues/2222)
-* Bugfix:  Upgrade Spring to version without vulnerability (#2252) ([f77fa1b](https://github.com/zowe/api-layer/commit/f77fa1b)), closes [#2252](https://github.com/zowe/api-layer/issues/2252)
-* Bugfix:  Use unique artifacts for gha (#2192) ([f8def8c](https://github.com/zowe/api-layer/commit/f8def8c)), closes [#2192](https://github.com/zowe/api-layer/issues/2192)
-* Bugfix:  Add x509 auth info to gw api doc (#2142) ([072ad23](https://github.com/zowe/api-layer/commit/072ad23)), closes [#2142](https://github.com/zowe/api-layer/issues/2142)
-* Bugfix:  Authorization provider set empty as default (#2107) ([aa77926](https://github.com/zowe/api-layer/commit/aa77926)), closes [#2107](https://github.com/zowe/api-layer/issues/2107)
-* Bugfix:  Change url for v2 and mark plugin as v2 in case very old apiml v1 copies don't support same url  ([6f4257a](https://github.com/zowe/api-layer/commit/6f4257a)), closes [#2022](https://github.com/zowe/api-layer/issues/2022)
-* Bugfix:  Update data model for infinispan storage in Caching service (#2156) ([38a1348](https://github.com/zowe/api-layer/commit/38a1348)), closes [#2156](https://github.com/zowe/api-layer/issues/2156)
+## `APIML 3.1.1 / Zowe 3.1.0 (2025-02-03)`
+
+* Feature:  Override external URL for additional registration (#3935) ([d5dd912](https://github.com/zowe/api-layer/commit/d5dd912)), closes [#3935](https://github.com/zowe/api-layer/issues/3935)
+* Feature:  Support OIDC token to authenticate in API Catalog (#3925) ([a4ead1d](https://github.com/zowe/api-layer/commit/a4ead1d)), closes [#3925](https://github.com/zowe/api-layer/issues/3925)
+* Feature:  Allows to obtain certificates from multiple sources (#3914) ([2e028cb](https://github.com/zowe/api-layer/commit/2e028cb)), closes [#3914](https://github.com/zowe/api-layer/issues/3914)
+* Feature:  Rate limit per service (#3903) ([cad63cb](https://github.com/zowe/api-layer/commit/cad63cb)), closes [#3903](https://github.com/zowe/api-layer/issues/3903)
+* Feature:  Add validate oidc token call to zaas client (#3897) ([3f0ac10](https://github.com/zowe/api-layer/commit/3f0ac10)), closes [#3897](https://github.com/zowe/api-layer/issues/3897)
+* Feature:  Limit API usage (#3868) ([bdbd3cb](https://github.com/zowe/api-layer/commit/bdbd3cb)), closes [#3868](https://github.com/zowe/api-layer/issues/3868)
+* Feature:  Java sample to authenticate with client certificate (#3862) ([992deb3](https://github.com/zowe/api-layer/commit/992deb3)), closes [#3862](https://github.com/zowe/api-layer/issues/3862)
```

Comment:
This is a simple application that showcases how do you build Java application that uses client certificate to authenticate against API Mediation Layer